.circleci/config.yml

```diff
@@ -37,6 +37,23 @@ jobs:
             # (optional) additional CI parameters to include in the pipeline (must be a valid JSON)
             # CI_PARAMETERS: '{ "deploy": true }'
 
+  trigger-tag:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - checkout
+      - run:
+          # See MarkKelnar about the personal api token created and added as an environment variable for the project
+          name: Set token
+          command: echo "export CIRCLE_USER_TOKEN=${CIRCLE_PROJECT_TOKEN}" >> $BASH_ENV
+      - run:
+          name: Trigger workflows with tag
+          command: chmod +x .circleci/monorepo.sh && .circleci/monorepo.sh
+          environment:
+            MONOREPO_DEBUG: "true"
+            # (optional) additional CI parameters to include in the pipeline (must be a valid JSON)
+            CI_PARAMETERS: '{ "wpe-headless-plugin": true }'
```

Comment:
We only have the one plugin/package at the moment that needs this behavior. Can come back and refine this to be more flexible but trying to get this out the door.